core/common/src/TimeZone.kt

```diff
@@ -12,7 +12,7 @@ import kotlinx.datetime.serializers.*
 import kotlinx.serialization.Serializable
 
 /**
- * A time zone, which is a collection of rules specifying which [LocalDateTime] values correspond to each [Instant].
+ * A time zone, which is a collection of rules specifying how to convert an [Instant] to a [LocalDateTime] and vice versa.
```

Comment:
"Vice versa" is not accurate if we assume that convertation is a function, which seems intuitive.

Reply:
My initial proposal:
> `A time zone, which is a collection of rules specifying which [LocalDateTime] values correspond to each [Instant].`

this doesn't say anything about which conversions are available, it only gives the intuition behind the data structure. We can also make a separate statement, like
```
[Instant.toLocalDateTime] is an unambiguous conversion according to these rules.
[LocalDateTime.toInstant] is a best-effort attempt at reversing that conversion.
```

Reply:
Currently, all that TimeZone exposes is the pair of conversion functions, not the collection of rules.

"which [LocalDateTime] values correspond to each [Instant]" implies that there could be several LDT values for a single Instant in the given TZ, which was my concern with it originally.

Reply:
Ah, ok. How about this then? `which [LocalDateTime] value correspond to each [Instant]`

Reply:
Ok, that would do. Rephrased to a middle ground between these two variants.